docs/wiki/99 - Developers/Automated Testing.md

```diff
@@ -91,7 +91,7 @@ We use Docker to help us simulate a server running LORIS. The installation and u
 
 LORIS uses the [PHPUnit library](https://phpunit.de/) for unit tests.
 
-Unit testing is covered in depth in our [Unit Test Guide](../../../test/UnitTestGuide.md).
+Unit testing is covered in depth in our [Unit Test Guide](https://github.com/aces/Loris/blob/master/test/UnitTestGuide.md).
```

Comment:
This should remain a relative link, what was wrong with the old one?

Reply:
I think that ReadTheDocs may not be able to travel outside of its root directory. I think this is the related to the issue #6209. We may have to put all of our documentation under `docs/wiki/` in order to serve it from ReadTheDocs.

Reply:
@johnsaigle Yes, ReadTheDocs is giving warning with all files outside the `docs/wiki` and I am not sure if using symlinks would pass ReadTheDocs since the log always tells me that the files could not be found for #6209, but I could move the documentation back to  `docs/wiki`.

Reply:
> This should remain a relative link, what was wrong with the old one?
It wasn't working, I also pointed it [here](https://neurostars.org/t/gsoc-2020-project-idea-24-loris-automated-testing/5763/7)

Reply:
I think the best approach here would be to move the Unit Test Guide to the folder `docs/wiki/99 - Developers/` and add a new entry in `mkdocs.yml`.

Reply:
@johnsaigle I could do that now. Could I also move the `LORIS-REST-API-0.0.2.md ` back to `docs` folder or create a copy inside the `docs/wiki` ? I believe this could resolve my PR #6276 for Issue #6209.

Reply:
That sounds good to me @haoweiqiu. Before you do that though let's wait for @driusan to weigh in.

Reply:
@johnsaigle Sure! However, I am not sure why we need to add new entry for the Unit Test Guide in `mkdocs.yml` since `Automated Testing.md` mentioned the Unit Test Guide already. Would you please explain a bit? Thanks!

Reply:
My thinking is that both documents, Unit Testing Guide and Automating Testing, should be in the developer section in the ReadTheDocs wiki. I don't think there's a good reason for both of these documents to be in different locations and I think the best location is in ReadTheDocs.

That way we can avoid the symlink/relative link issue and keep everything in the `docs/wiki/...` folder which should make things more simple.

Is that more clear? let me know if I understood you correctly.

Reply:
We should not be moving things out of their modules just for read the docs, if anything we should just move the read the docs config up a level so that it can access anything in the repo.

Reply:
@johnsaigle Thanks I understand now.

Reply:
@driusan Thanks for your comment! I would continue to find solution for #6209.